lib/i18n/en_US.json

```
@@ -75,7 +75,7 @@
 	"please_upload_all_documents": "Upload All Docs",
 	"Refund_request_submitted_successfully": "Refund Submitted",
 	"View_Document": "View Doc",
-	"file_uploaded": "Uploaded",
+	"file_uploaded": "Uploaded ",
```

space should be removed.
```suggestion
	"file_uploaded": "Uploaded",
```